### ISSUE TITLE
New output cell for each loop

### ISSUE THREAD

**Issue by bcikili** [microsoft/vscode-jupyter#11365]
### Applies To

- [X] Notebooks (.ipynb files)
- [ ] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

### What happened?

![jupy](https://user-images.githubusercontent.com/35443804/189995941-16fe3e77-ee44-4a7a-8ce2-82c4ee824677.jpg)

For each for loop it creates a new output cell.  if output log is long it turn gray and can not read or see the logs. I believe it also make jupyter slow in response time. 

### VS Code Version

1.70.2

### Jupyter Extension Version

v2022.7.1102252217

### Jupyter logs

```shell
added photo in 'What Happened?'
```

### Coding Language and Runtime Version

Python 3.9

### Running Jupyter locally or remotely?

Local

**Comment by roblourens** [comment]
I don't understand the issue. Please give some example code and clear description of what the problem is.

**Comment by andycraig** [comment]
Hi, I believe I experienced the same problem and somehow resolved it so I'm sharing some notes. For me this was all when using the Interactive Window. I don't use the Notebook interface.

In my case the issue started around two weeks ago. I wasn't aware of any particular change that occurred at that point - I don't remember installing any new extensions, for example.

Normally when running a single chunk of code, the output all appears in a single Interactive Window output cell. But with this issue, printing a new line seemed to sometimes create a new Interactive Window output cell, that included the contents of the previous cell plus the new line. This can be seen in the original post.

It seemed to be somewhat speed dependent. That is, if I ran a cell like this

```py
for i in range(10):
    print(i)
```

it WOULD appear in the same cell. But if the operation being looped over was slower, it would create a new cell each iteration.

There were other display issues too. For example, running code consisting of just a list (e.g.,  `[1, 2, 3]`) would produce a large blank cell with no content.

**How I resolved it**
1. I clicked the '...' to the left of the cell and chose 'Change presentation'
    This opened a dialog 'Select mimetype to render for current output', with the following choices:
    - `text/plain VS Code Builtin Notebook Output Renderer (vscode.builtin-notebook-renderers)`
    - `text/plain Data Table (RandomFractalsInc.vscode-data-table)`
    - `text/plain Data Summary (RandomFractalsInc.vscode-data-table)`
    - `text/plain Flat Data Grid (RandomFractalsInc.vscode-data-table)`
1. The selected option at that time was one of the `RandomFractalsInc.vscode-data-table` options. (I can't remember which.) I selected the `vscode.builting-notebook-renderers` option.
1. _After doing this for several cells_, the `vscode.builting-notebook-renderers` option seemed to become the default, and from that point I haven't experienced the issue again.

In summary, I the issue I encountered, which I believe is probably the same issue described above, is that the default mimetype suddenly changes with no notification to the user. It seems that the default mimetype can then be changed back, but it isn't obvious to the user what they need to do. (In my case it took a week or so before I randomly stumbled across the resolution.)

Hope that helps!

**Comment by bcikili** [comment]
it worked, thank you  andycraig .
Let's state the problem here again, after updating VS Code, it changed presentation preference without any notification, since I ve never used this type of output format never thought it is related to mimetype.

**Comment by jslomkowski** [comment]
Hi, if anyone wonders, this is caused by the "Data Table Renderers" extension
